src/Aspire.Hosting.ValKey/Aspire.Hosting.ValKey.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageTags>aspire hosting valkey</PackageTags>
+    <Description>Valkey® support for .NET Aspire.</Description>
+    <PackageIconFullPath>$(SharedDir)valkey-icon.png</PackageIconFullPath>
```

Comment:
Do we have permission to reuse the icon?

Reply:
I didn't ask for permission
Remove